src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsEndpoint.java

```diff
@@ -1,22 +1,11 @@
 package org.jboss.aerogear.keycloak.metrics;
 
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.*;
```

Comment:
why switching to *-imports? makes maintenance worse

Reply:
That's just what IntelliJ made of it. Hadn't noticed it.
Replaced with single class imports.